features/admin.py

```diff
@@ -19,6 +20,10 @@
 )
 
 
+def hide_feature(modeladmin, request, queryset):
```

Comment:
what's the reason for keeping it outside of the class?

Reply:
should it have a description? `hide_feature.short_description = "This hides stuff"`?

Reply:
Moved the method inside the class.
I don't think it needs a separate description as it does exactly what the function name states - hides features.